packages/components/src/core/SegmentedControl/index.tsx

```diff
@@ -57,16 +58,16 @@ const SegmentedControl = (props: SegmentedControlProps) => {
         };
 
         return (
-          <ToggleButton
-            aria-label={tooltipText}
-            disableRipple
-            value={value}
+          <Tooltip
+            title={tooltipText ?? value}
+            sdsStyle="dark"
+            arrow
             key={value}
           >
-            <Tooltip title={tooltipText ?? value} sdsStyle="dark" arrow>
-              <span>{iconItem()}</span>
-            </Tooltip>
-          </ToggleButton>
+            <ToggleButton aria-label={tooltipText} disableRipple value={value}>
```

Comment:
I thought `ripple` was disabled for all components already. Maybe in a follow up PR we can see if we can add it in the theme option 😄 ?

https://github.com/chanzuckerberg/sci-components/blob/96a2af14747a5413c1bbd45ad45c35a10d52f76d/packages/components/src/core/styles/common/makeThemeOptions.ts#L12

Reply:
Oh, wow! I have to say it, you have an eagle eye for bugs and improvements 🦅 
Your attention to detail is impressive! 😯🤩🥳🎉
You're right. We don't need it here. All the ripple effect are disabled by default. 🙏🏻

Reply:
Oh yay glad to hear that! Thank you for double checking 😄 🙏 Ahaha you're too kind 🙇 You also spot many things I missed in my PRs. We make a great team 🔥 🕺 🕺